### PR TITLE
Recent acitivities

### PR DESCRIPTION
**Pull request by shan-57blocks**

### CODE REVIEW

**Review by PlayJok3r — APPROVED** [review]

File: packages/huma-shared/src/solana/metadata/devnet.ts [hunk 1]
```diff
@@ -32,5 +32,6 @@ export const SOLANA_DEVNET_METADATA: SolanaPoolsInfo = {
       decimals: 6,
       icon: 'USDC',
     },
+    receivable: '5o7qiQZeCbTowg75xgB5xMnYRHpXx43c6CnFbnC3MXkJ',
```
Comment (mliu) [inline packages/huma-shared/src/solana/metadata/devnet.ts:35]: super nit, can we move this above the underlyingMint?

File: packages/huma-shared/src/solana/pool.ts [hunk 2]
```diff
@@ -26,6 +26,8 @@ export type SolanaPoolInfo = {
     decimals: number
     icon: string
   }
+  receivable?: string
```
Comment (mliu) [inline packages/huma-shared/src/solana/pool.ts:29]: ditto

File: packages/huma-shared/src/solana/pool.ts [hunk 3]
```diff
@@ -26,6 +26,8 @@ export type SolanaPoolInfo = {
...
+  isClosed?: boolean
```
Comment (mliu) [inline packages/huma-shared/src/solana/pool.ts:30]: can we move isClosed under an `extra` mapping?

File: packages/huma-shared/src/solana/metadata/devnet.ts (hunk 1 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/solana/metadata/devnet.ts:35]: Updated in a later PR

File: packages/huma-shared/src/solana/pool.ts (hunk 2 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/solana/pool.ts:29]: Updated in a later PR

File: packages/huma-shared/src/solana/pool.ts (hunk 3 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/solana/pool.ts:30]: Updated in a later PR

**Pull request merged**